Game/Object/Human.cs

```diff
@@ -38,17 +38,26 @@ public class Pedestrian : Human, IObjectToFollow
         public bool IsEnteringVehicle => _Is(CitizenInstance.Flags.EnteringVehicle);
         public bool IsHangingAround => _Is(CitizenInstance.Flags.HangAround);
 
+        public uint GetTargetFrame()
```

Comment:
isn't this just
```csharp
public uint GetTargetFrame()
    => SimulationManager.instance.m_referenceFrameIndex - id._index >> 12;
```
Maybe I miss something?

Reply:
I assume you meant `<< 12`.
maybe it is different in terms of rounding?
I just copied this from decompiler (`CitizenInstance.GetSmoothPosition`). not sure why it is like this. but the calculations are very sensitive. slightest mismatch can cause the red look arrow to jump forward and backward at the end of the frame interval.

Reply:
ill ask in the discord forum

Reply:
BTW I have not got around completing the HUMAN look ahead calculations yet. the vehicle is the one you need to check first.

Reply:
I edited the message.

Reply:
Actually I cannot see any difference https://dotnetfiddle.net/Widget/PlHmKB

Reply:
got my answer https://discord.com/channels/436608207788703753/436616431493447682/979825332574093312